### PR TITLE
Added swipe capability to gestures

### PR DESCRIPTION
**Pull request by dpgraham**
Called via execution of 'mobile: swipe'. Must provide direction (up, down, left, or right). Similar to 'scroll'.

### CODE REVIEW

**Comment by imurchie** [comment]
@jlipps Do we want to re-introduce `mobile: xxx` commands? It would make life easier (not having to try to map the Touch Actions). If WDA _does_ implement arbitrary actions in the future we would have to revamp the API anyway.

**Comment by jlipps** [comment]
@imurchie dan and i had a long convo about this yesterday. Ultimately this version of swipe from WDA is not at all related to any touch action API. It basically does a flick with a constant speed every single time (essentially a snappier version of WDA's scroll endpoint). So it doesn't map to any of the existing APIs on our end. What I hear from @marekcirkos is that he's interested in supporting the full TouchAction API. In light of that, I thought it best to just resurrect `mobile: swipe` since the machinery is all there, instead of creating a new endpoint which will become obsolete in short order.

**Comment by dpgraham** [comment]
I just made the changes you recommended and squashed them into one commit.

File: test/functional/basic/gesture-e2e-specs.js [hunk 1]
```diff
@@ -113,6 +114,15 @@ describe('XCUITestDriver - gestures', function () {
 
       await driver.elementByAccessibilityId('2').should.not.be.rejected;
     });
+    it('should swipe the table and the bottom cell\'s Y position should change accordingly', async () => {
```
Comment (imurchie) [inline test/functional/basic/gesture-e2e-specs.js]: Not a material change, but an observation: if you use backticks you don't need to escape the apostrophe!

**Pull request merged**